cla_public/apps/base/filters.py

```diff
@@ -14,3 +16,20 @@ def datetime(dt, format='medium', locale='en_GB'):
     elif format == 'short':
         format = "dd/MM/y, h:mma"
     return format_datetime(dt, format, locale=locale)
+
+
+@base.app_template_filter()
+def url_to_human(value, trim_url_limit=None, nofollow=None, target=None):
+    return re.sub(r'(^https?://)|(/$)', '', value)
+
+@base.app_template_filter()
+def human_to_url(value, trim_url_limit=None, nofollow=None, target=None):
+    return re.sub(r'^((?!https?://).*)', r'http://\1', value)
```

Comment:
These functions both have unused parameters `trim_url_limit`, `nofollow` and `target`